app/ds_config_sample.py

```diff
@@ -30,6 +30,6 @@
 DS_JWT = {
     "ds_client_id": "{CLIENT_ID}",
     "ds_impersonated_user_id": "{IMPERSONATED_USER_ID}",  # the id of the user
-    "private_key_file": "private.key", # path to file which hold private key or private key itself
+    "private_key_file": "./private.key", # create a new file named private.key in the app directory
```

Comment:
"Create a new file in your repo source folder named **private.key** then copy and paste your RSA private there and save it."